RELEASE_NOTES.md

```diff
@@ -4,26 +4,135 @@
+
+`Enabled by default`: True
+
+`Autocorrects`: No
+
 #### Chef/WindowsVersionHelper
 
+The `WindowsVersionHelper` cop detects the usage of the various Windows version helpers that shipped in the [windows](https://supermarket.chef.io/cookbooks/windows) cookbook. Nearly all of the resources that originally shipped in Windows cookbook are now part of Chef Infra Client itself. We'd like to help move users away from this cookbook so they can avoid the added dependency in their environment. Instead of using the Windows version helpers we suggest using data from Ohai itself to determine what version of Windows is running on a node.
+
+`Examples`
+
+A cookbook using the Windows version helper:
+```ruby
+  Windows::VersionHelper.nt_version == 6.1
+```
+
+The same version comparison could be done using Ohai attributes
+```ruby
+  node['platform_version'].to_f == 6.1
+```
+
+`Enabled by default`: True
+
+`Autocorrects`: No
+
 #### Chef/WindowsZipfileUsage
 
+The `WindowsZipfileUsage` detects the usage of the `windows_zipfile` resource in the [windows](https://supermarket.chef.io/cookbooks/windows) cookbook. This is one of the remaining resources in the windows cookbook that is not present in Chef Infra Client itself. In Chef Infra Client 15.0 we introduced the [archive_file](https://docs.chef.io/resource_archive_file.html) resource, which is a cross platform resource for uncompressing archives. This resource uses an ulta-fast compression library with support for multiple compression formats, and should be used instead of the legacy `windows_zipfile` resource.
+
+`Enabled by default`: True
+
+`Autocorrects`: No
+
 #### Chef/NodeMethodsInsteadofAttributes
 
+The `NodeMethodsInsteadofAttributes` cop detects cookbooks that reference node attributes in the legacy `node.some.value` format. Chef Infra Client 13 officially deprecated this attribute format. This cop detects the usage of common Ohai attributes and can autocorrect those, but it won't detect cookbook attributes, which will also need to be updated.
+
+`Enabled by default`: True
+
+`Autocorrects`: Yes
+
+`Examples`
+
+Node attributes in the method format:
+```ruby
+  node.platform
+  node.platform_version
+  node.my_cookbook.attribute1
+```
+
+The preferred format:
+```ruby
+  node['platform']
+  node['platform_version']
+  node['my_cookbook']['attribute1']
+```
+
 #### Chef/IncludingMixinShelloutInResources
 
+The `IncludingMixinShelloutInResources` cop detects resources that require and include the Chef Shellout Mixin helper. Chef Infra Client now includes this mixin helper automatically for all resources so this require and include can be removed.
```

Comment:
Are there other words supposed to be used instead of "require and include in the last sentence? ("this require and include can be removed."

Reply:
require and include are the actual Ruby calls that are being called